ner_v1/chatbot/entity_detection.py

```diff
@@ -565,13 +577,18 @@ def get_person_name(message, entity_name, structured_value, fallback_value, bot_
     entity_list, original_text_list = [], []
 
     if text:
-        entity_list, original_text_list = name_detection.detect_entity(text=text, bot_message=bot_message)
+        entity_list, original_text_list = name_detection.detect_entity(
+            text=text,
+            bot_message=bot_message,
+            free_text_detection_results=free_text_detection_results)
 
     if not entity_list and fallback_text:
         entity_list, original_text_list = NameDetector.get_format_name(fallback_text.split(), fallback_text)
         detection_method = fallback_method
 
     if entity_list and original_text_list:
+        if free_text_detection_results:
+            detection_method = "free text entity"
```

Comment:
If possible can we follow the same convention as text detector's output and add `DATASTORE_VERIFIED` and `CRF_MODEL_VERIFIED`

Reply:
Done